ckanext/xloader/utils.py

```diff
@@ -256,3 +274,15 @@ def datastore_resource_exists(resource_id):
     except p.toolkit.ObjectNotFound:
         return False
     return response or {'fields': []}
+
+
+def is_validation_plugin_loaded():
+    """
+    Checks the existence of a logic action from the ckanext-validation
+    plugin, thus supporting any extending of the Validation Plugin class.
+    """
+    try:
+        p.toolkit.get_action('resource_validation_show')
```

Comment:
so we are expecting to have validation plugin loaded and that this action is available
https://github.com/frictionlessdata/ckanext-validation/blob/master/ckanext/validation/plugin/__init__.py#L87

Reply:
Yeah, Thrawn mentioned to do this, as there are cases in which developers could sub-class the Validation Plugin class, and rename the entry-point plugin name.

I personally have done this for at least two plugins in our setup. So was a good idea I think